### PR TITLE
WIP - Update Custom WAR Packager to 1.3 and to Mainstream Jenkinsfile Runner

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Downstream of https://github.com/jenkinsci/custom-war-packager/pull/44. Updates JCasC to the released version and replaces Jenkinsfile Runner fork by the official upstream version.

Work in progress, CWP still needs a release (I hope to have a CD flow in that repository soon)

### CODE REVIEW

**Comment by jglick** [comment]
How does this relate to #52?

File: packager-config.yml [hunk 1]
```diff
@@ -14,9 +14,8 @@ buildSettings:
         #TODO: replace once Jenkinsfile Runner archives runner & CWP is updated
         noCache: true
       source:
-        #dir: "/Users/nenashev/Documents/jenkins/tools/jenkinsfile-runner"
-        git: https://github.com/kohsuke/jenkinsfile-runner.git
-        commit: 8ff9b1e9a097e629c5fbffca9a3d69750097ecc4
+        git: https://github.com/jenkins/jenkinsfile-runner.git
+        commit: 91cdf24849c4f7367322ac511574d294c5882c2f
```
[the remark this answers is not in the document]
Comment (oleg-nenashev) [reply packager-config.yml]: With the current design it does not make much sense to release it in a common way - you have to rebuild it with the core.  But yes, here is https://issues.jenkins-ci.org/browse/JENKINS-54375 for it

**Comment by oleg-nenashev** [comment]
> How does this relate to #52?

It subsumes it

**Comment by oleg-nenashev** [comment]
Closing in favor of #52

**Pull request closed without merging**